cypress/integration/thermometer.js

```diff
@@ -0,0 +1,51 @@
+context("Test the Thermometer Icon", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("thermometer at bottom bar", () => {
+    cy.get(".icon-button--iconButton--__hurr-v1__").should("be.visible");
+  });
+
+  it("thermometer enable after run", () => {
+      cy.get('[data-test="start-button"]').should("be.visible");
+      cy.get('[data-test="start-button"]').click();
+      cy.wait(500).then(() => {
+        cy.get('[data-test="start-button"]').click();
+
+      });
+      cy.get(".icon-button--iconButton--__hurr-v1__").should("be.visible");
```

Comment:
.icon-button is a generic name of a component. At this point, it's only a thermometer, but we might reuse this component for more buttons. I think you should be able to use `[data-test="temp-button"]`.

Reply:
Updated